### ISSUE TITLE
unlink bug?

### ISSUE THREAD

**Issue by jcerdan** [sparkida/ftpimp#25]
Hi,

I was having a bug using unlink function. I had to modify line 1424 to this

`data = data.match(/DELE (.*)/)[1];`

could you check please?

By the way, nice job ;)

**Comment by sparkida** [comment]
Thanks for the info @jcerdan, knowing that there are differences here has me concerned as to how many differences remain undiscovered. This issue is because I develop against a [Pure-FTPd](https://www.pureftpd.org/project/pure-ftpd) server which sends the response of *"Deleted myfile.txt"*. In order to make this more portable I've added `/(del\w+) *(.*)/i` in #26 and created ticket #27 for adding different ftp builds to Travis.

***TLDR;***
[3.0.2](https://www.npmjs.com/package/ftpimp) was released

Thanks again!
Nick
